lib/glific.ex

```diff
@@ -25,6 +27,29 @@ defmodule Glific do
     end
   end
 
+  @doc """
+  Validates inputed shortcode, if shortcode is invalid it returns message that the shortcode is invalid
+  along with the valid shortcode.
+  """
+  @spec(
+    validate_shortcode(Ecto.Changeset.t()) :: Ecto.Changeset.t() | Ecto.Changeset.t(),
+    atom(),
+    String.t()
+  )
+  def validate_shortcode(%Ecto.Changeset{} = changeset) do
+    shortcode = Map.get(changeset.changes, :shortcode)
+    valid_shortcode = Glific.string_clean(shortcode)
+
+    if string_clean(shortcode) == shortcode,
```

Comment:
shouldnt this be valid_shortcode == shortcode

Ultimately, i think the frontend will automatically create the shortcode and allow the user to overwrite it

Reply:
Yea, it's miss